sdk/eventhub/event-hubs/src/eventProcessor.ts

```diff
@@ -572,6 +571,24 @@ function isAbandoned(ownership: PartitionOwnership): boolean {
   return ownership.ownerId === "";
 }
 
+function getFallbackPosition(partitionIdToClaim: string, fallbackPositions?: EventPosition | Map<string, EventPosition>) : EventPosition {
+  if (fallbackPositions == null) {
+    return EventPosition.earliest();
```

Comment:
.Net and Java use earliest as the default for the getting started scenario, but use latest for the EventProcessorClient case

Python & JS use the same client & function for all cases. We should probably default to latest.

Reply:
cc @YijunXieMS and @chradek

Reply:
Agreed, will switch to latest.